### PR TITLE
add cpanato as Technicians

### PR DESCRIPTION
**Pull request by cpanato**
/assign @alexellis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,26 +80,27 @@ See the new [growlab app](/app) for your Raspberry Pi
 
 | Name  | Bronze   | Silver   | Gold   | Platinum |
```
Comment (alexellis) [inline README.md:81]: What happened to this section?

Comment (cpanato) [reply README.md:81]: removed some spaces and did some alignment in the code, I can revert

Comment (alexellis) [reply README.md:81]: It would be easier to merge if you just added your line(s) yep thanks

Comment (cpanato) [reply README.md]: done

**Comment by alexellis** [comment]
Thank you

**Pull request merged**